### PR TITLE
Add filled icons for global nav use cases (Issue/2444)

### PR DESCRIPTION
**Pull request by tatjana-gartner2**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://github.com/porscheofficial/oss-docs/blob/67c5b24838a293ce7a964884e6005eb71f2b8579/CONTRIBUTOR_LICENSE_AGREEMENT.md)

#### References

- Preview: https://designsystem.porsche.com/issue/.../

#### Scope

Short description of the pull request scope

### CODE REVIEW

File: packages/components/package.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
     "build:wrappers": "yarn build:wrappers:generateWrappers && yarn build:wrappers:generateDSRComponents && yarn build:wrappers:bundleStyles && yarn build:wrappers:bundleUtils",
     "build:wrappers:generateWrappers": "ts-node --project tsconfig.node.json scripts/generateWrappers.ts",
     "build:wrappers:generateDSRComponents": "ts-node --project tsconfig.node.json scripts/generateDSRComponents.ts",
-    "build:wrappers:bundleStyles": "rm -rf ./src/styles-entry.ts && rollup -c rollup-styles.config.js --bundleConfigAsCjs",
-    "build:wrappers:bundleUtils": "rm -rf ./src/utils-entry.ts && rollup -c rollup-utils.config.js --bundleConfigAsCjs",
+    "build:wrappers:bundleStyles": "rm -rf ./src/styles-entry.ts && PDS_IS_STAGING=${PDS_IS_STAGING:-1} rollup -c rollup-styles.config.js --bundleConfigAsCjs",
```
Comment (stephanschroeter) [inline packages/components/package.json:20]: Why is this needed?

File: packages/components-react/projects/react-ssr-wrapper/tests/unit/specs/package.spec.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import * as path from 'path';
+import * as globby from 'globby';
+import * as fs from 'fs';
+
+describe('package content', () => {
+  const componentsReactFilePath = require.resolve('@porsche-design-system/components-react');
+  const componentsReactPackageDir = path.resolve(componentsReactFilePath, '..');
+  const componentsReactFilePaths = globby.sync(`${componentsReactPackageDir}/**/*.js`);
```
Comment (denyo) [inline packages/components-react/projects/react-ssr-wrapper/tests/unit/specs/package.spec.ts:8]: Wouldn't the `ssr` subpackage be more precise to check?

File: packages/components-react/projects/react-ssr-wrapper/tests/unit/specs/__snapshots__/react-ssr-wrapper.spec.tsx.snap [hunk 3]
```diff
@@ -1268,8 +1268,11 @@ button::before {
   right: -4px;
   top: 6px;
   bottom: 6px;
+  transition: background-color var(--p-transition-duration, .24s) ease;
```
Comment (denyo) [inline packages/components-react/projects/react-ssr-wrapper/tests/unit/specs/__snapshots__/react-ssr-wrapper.spec.tsx.snap:1271]: We shouldn't output any hover styles no server side since they are unnecessary for the first split seconds of page load.  
I guess the `stripFocusAndHoverStyles` utility needs to be extended to handle both, the `build` and `build-prod` scenario.

File: packages/components/package.json [hunk 4]
```diff
@@ -17,8 +17,8 @@
     "build:wrappers": "yarn build:wrappers:generateWrappers && yarn build:wrappers:generateDSRComponents && yarn build:wrappers:bundleStyles && yarn build:wrappers:bundleUtils",
```
Comment (denyo) [inline packages/components/package.json:17]: Looks like changelog entry is missing.  
Once added, you can update storefront's `sitemap.json` with the url to this PR, too.

**Review by denyo — APPROVED** [review]
